src/bin/polymer-bundler.ts

```diff
@@ -172,8 +171,10 @@ const usage = [
 const options = commandLineArgs(optionDefinitions);
 const projectRoot =
     options.root ? pathLib.resolve(options.root) : pathLib.resolve('.');
+const projectRootUrl =
+    ensureTrailingSlash(Uri.file(projectRoot).toString()) as ResolvedUrl;
```

Comment:
consider moving to a named function

Reply:
Done.  Created `getFileUrl` to dry it up.  Also created an analog to `path.resolve()` called `resolvePath` which preserves trailing slashes.  Converted all the places I call `ensureTrailingSlash`, `path.resolve()` and `Uri.file` to use the functions where possible.